Snowflake.Data/Core/HttpUtil.cs

```diff
@@ -517,6 +515,13 @@ protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage
             }
         }
 
+        static private Exception GetInnerMostException(Exception innermostException)

```

Comment:
I know that it works, but someone who reads it could be confused to see innermostException as a parameter name because it should be the result, not the input parameter.
Maybe we could rename the parameter `GetInnerMostException(Exception exception)` and then start the method with `var innermostException = exception;`

Reply:
Done